.github/workflows/rename-pr.yml

```diff
@@ -0,0 +1,21 @@
+name: Rename PR Title
```

Comment:
can this be renamed to "Rename Konflux PR Title"? just to make it more specific

Reply:
OK, done, thank you